src/core/public/index.ts

```diff
@@ -50,7 +50,6 @@ import './index.scss';
 
 import {
   ChromeBadge,
-  ChromeBrand,
```

Comment:
This is a change to public API, if I'm not mistaken, is that something we care about in this context?